### ISSUE TITLE
Colander: controller-1 not synchronising after passing controll to controller-2

### ISSUE THREAD

**Issue by Marc69** [clustervision/trinity#490]
**Version:** r6 6859708
**Environment:** unknown

[root@controller-1(unknown*) ~]# cat /proc/drbd 
version: 8.4.6 (api:1/proto:86-101)
GIT-hash: 833d830e0152d1e457fa7856e71e11248ccf3f70 build by phil@Build64R7, 2015-04-10 05:13:52

**Comment by hansthen** [comment]
Is this a duplicate of #651?

**Comment by Marc69** [comment]
I think this was only after failover. #651 is before failover.
